x/nft/username/model_test.go

```diff
@@ -88,7 +88,8 @@ func TestChainAddressValidation(t *testing.T) {
 		{chainID: "1234", address: "", expError: true}, // empty address
 		{chainID: "1234", address: "", expError: true},
 		{chainID: "", address: "123456789012", expError: true},
-		{chainID: "1234", address: "12345678901", expError: true},
+		{chainID: "1234", address: "1", expError: true},
+		{chainID: "1234", address: "12", expError: false},
```

Comment:
Why not use "1L" as an example to reminde future you why such short addresses are okay?

Reply:
Where `1L` comes from? I would like to put this in the comment, so that it's not a random value.

Reply:
Lisk uses `<number>L` with number being any uint64 number represented as decimal. `0L`, `1L`, ..., `9L` are the shortest valid Lisk addresses.  [Average addresses](https://testnet-explorer.lisk.io/) are longer.